data/autoyast_sle15/create_hdd/create_hdd_sap.xml.ep

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <install_updates config:type="boolean">true</install_updates>
+    <slp_discovery t="boolean">false</slp_discovery>
+    % if ($check_var->('SLE_PRODUCT', 'sles4sap')) {
```

Comment:
Do we not need these conditions, as this profile is only for SAP ? Otherwise if the profile is to be used also for other products, I would call it just create_hdd.xml.ep.

Reply:
good point to remove the condition, but let's keep specific profile for sap in the naming, otherwise with generic name might not be clear what is the use case of it.

In the future I hope we could have all our autoyast profile in one yast/ folder so we can prevent changes from other contributors using this profiles, as it doesn't make sense to reuse, because each squad has specific needs for them.

Reply:
I have removed the conditions, please check